step1/src/app.js

```diff
@@ -0,0 +1,104 @@
+import ItemAppender from "../components/ItemAppender";
+import Items from "../components/Items";
+import ItemModifier from "../components/ItemModifier";
+import Component from "../core/Component";
+
+
+
+
+export default class App extends Component{
+  
+    setup() {
+        this.$state = {
+            todoItems: [
+                {
+                    id: 1,
+                    content: `item1`,
+                    isComplete: false,
+                },
+                {
+                    id: 2,
+                    content: `item2`,
+                    isComplete: false,
+                }
+            ],
+            selectedItem:-1
+        }
+    }
+    template() {
+        return `
+        <h1>📃 TodoList</h1>
+        <form data-component="appenderForm" action="" method="post"></form>
+        <ul data-component="modifierForm items"></ul>
+        `;
+    }
```

Comment:
```suggestion
    template() {
        return `
          <h1>📃 TodoList</h1>
          <form data-component="appenderForm" action="" method="post"></form>
          <ul data-component="modifierForm items"></ul>
        `;
    }
```
이렇게 indent를 한 칸 늘려주세요!